x-pack/plugins/ml/public/application/jobs/jobs_list/components/utils.js

```diff
@@ -76,6 +77,10 @@ export function isClosable(jobs) {
   );
 }
 
+export function isResetable(jobs) {
```

Comment:
Nit - this should be `isResettable` I think (consistent with the existing `isStoppable`)